js/form.js

```diff
@@ -0,0 +1,197 @@
+import { elements } from "./variables";
```

Comment:
better to split form module into sub module. Try to have very well defined responsibility for each module.
- like validation module should just do validation.
- loading module should just do loading etc etc.